students/wielgus_daniel/lesson_01_basics/lesson_01_part_01.py

```diff
@@ -0,0 +1,78 @@
+import math
+
+#Problem1: Sum of numbers
+print("We will count the Sum of three numbers")
+print("Please write first number")
+firstNumber = int(input())
+print("Your choice is: " + str(firstNumber))
+print("Please write second number")
+secondNumber = int(input())
+print("Your choice is: " + str(secondNumber))
+print("Please write third number")
+thirdNumber = int(input())
+print("Your choice is: " + str(thirdNumber))
+print("Your choose numbers: " + str(firstNumber) + ", " + str(secondNumber) + ", " + str(thirdNumber))
+sum = firstNumber + secondNumber + thirdNumber
+print("The sum of yours numbers is: " + str(int(sum)))
+print("THE END of Sum numbers problem")
+print("")
+
+#Problem2: Area of right-angled triangle
+print("We will count the area of right-angled triangle")
+print("Please write length of the base:")
+base = int(input())
+print("Please write the high of right-angle triangle:")
+high = int(input())
+area = (base * high)//2
```

Comment:
using `//` operator is a floor division. So area will be integer, with lost fraction part.
Output from this code is wrong. For input `3 5` the output should be `7.5`, where this code produces `7`

Please make sure that code passes all tests from https://snakify.org/lessons/print_input_numbers/problems/area_of_right_triangle/

Please do same for all future snakify problems